elliottlib/errata_async.py

```diff
@@ -91,51 +92,114 @@ async def delete_cve_package_exclusion(self, exclusion_id: int):
 
 class AsyncErrataUtils:
     @classmethod
-    async def get_advisory_cve_package_exclusions(cls, api: AsyncErrataAPI, advisory_id: int):
+    async def get_advisory_cve_exclusions(cls, api: AsyncErrataAPI, advisory_id: int):
```

Comment:
Var names here are huge and the extra words are not adding to their meaning/intent. Keep them succint